lib/loadGltfUris.js

```diff
@@ -115,9 +115,10 @@ function isTransparent(image) {
 function readImageFromFile(object, name, uriPath) {
     return fsReadFile(uriPath)
         .then(function(data) {
+            var ext = path.extname(uriPath);
             object.extras._pipeline.source = data;
-            object.extras._pipeline.extension = path.extname(uriPath);
-            if (name === 'images') {
+            object.extras._pipeline.extension = ext;
+            if (name === 'images' && ext !== '.ktx' && ext !== '.gif') {
```

Comment:
Like this file with an embedded gif - Jimp will try to load it but fail: 
[BoxTextured-optimized.gltf.txt](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/files/662235/BoxTextured-optimized.gltf.txt)